src/errors.rs

```diff
@@ -55,6 +67,10 @@ pub enum VMError {
     #[fail(display = "Item is not a wide value.")]
     TypeNotWideValue,
 
+    /// This error occurs when an instruction expects a key type.
+    #[fail(display = "Item is not a key value.")]
+    TypeNotKeyValue,
```

Comment:
`TypeNotKey` is more accurate: there is nothing about Value. Same for the display msg and a comment.